src/Compilers/Test/Resources/Core/CompilerTestResources.csproj

```diff
@@ -117,6 +117,12 @@
     <EmbeddedResource Include="MetadataTests\Invalid\ManyMethodSpecs.vb" />
     <EmbeddedResource Include="MetadataTests\Invalid\Obfuscated.dll" />
     <EmbeddedResource Include="MetadataTests\Invalid\Obfuscated2.dll" />
+    <Content Include="MetadataTests\Invalid\Signatures\build.bat" />
+    <Content Include="MetadataTests\Invalid\Signatures\munge.csx" />
+    <Content Include="MetadataTests\Invalid\Signatures\SignatureCycle2.il" />
+    <Content Include="MetadataTests\Invalid\Signatures\TypeSpecInWrongPlace.il" />
+    <EmbeddedResource Include="MetadataTests\Invalid\Signatures\SignatureCycle2.exe" />
```

Comment:
It is tiny. This is the current pattern we use in compilers, I don't see what's wrong with it. The file will never change.